docs/decisions/0003-django-setting-toggles.rst

```diff
@@ -32,41 +32,29 @@ We will introduce a simple wrapping class (or classes) that can be used to expli
 Here are some example instances::
 
     # Toggle setting
-    EXAMPLE_SETTING_TOGGLE = DjangoSettingToggle("EXAMPLE_TOGGLE", default=False)
+    EXAMPLE_SETTING_TOGGLE = SettingToggle("EXAMPLE_TOGGLE", default=False)
 
     # or
 
-    # Toggle setting in a dict: Option 1
-    EXAMPLE_SETTING_TOGGLE = DjangoSettingToggle(
-        "EXAMPLE_SETTING_DICT['EXAMPLE_TOGGLE']",
-        default=False,
-    )
-
-    # or
-
-    # Toggle setting in a dict: Option 2
-    EXAMPLE_SETTING_TOGGLE = DjangoSettingDictToggle(
+    # Toggle setting in a dict
+    EXAMPLE_SETTING_TOGGLE = SettingDictToggle(
         'EXAMPLE_SETTING_DICT',
         'EXAMPLE_TOGGLE',
         default=False,
     )
 
-    # TODO: Before implementing, we'll need to choose between Option 1 and Option 2 for
-    #   toggle setting in a dict. We can update this decision once this has been decided,
-    #   and move the rejected option, explaining why.
-
 It's usage can now match that of the waffle wrapping classes (e.g. WaffleFlag).
 
 Here is an example usage::
 
     EXAMPLE_SETTING_TOGGLE.is_enabled()
 
-DjangoSettingToggle will allow for expicit toggle definitions, with the following benefits:
+SettingToggle will allow for explicit toggle definitions, with the following benefits:
 
 * The definition can live with the app where it will be used, rather than in `common.py` or with no explicit definition.
 * The explicit definition can be annotated with documentation.
 * The explicit definition provides a single place to set a consistent default value.
```

Comment:
Where should that location be, though? There may be several locations of use that are not "near" each other in the code, so we will likely still end up with a common-file situation.

Reply:
For waffle flags we had an undocumented pattern of adding a `waffle.py` file to an app.  I think we should instead have a `toggles.py`, and they all could live there.

Reply:
I can add an update around this to the ADR if you'd like.

Reply:
@timmc-edx: FYI: I added an update for this.